indexing-service/src/main/java/org/apache/druid/indexing/common/actions/UpdateStatusAction.java

```diff
@@ -34,13 +34,24 @@ public class UpdateStatusAction implements TaskAction<Void>
 {
   @JsonIgnore
   private final String status;
+  @JsonIgnore
+  private final TaskStatus statusFull;
+
+  public UpdateStatusAction(
```

Comment:
Should this be marked as deprecated so it can be removed in a future version. I think it's still being used in some unit tests.